l10n/en/brands.ftl

```diff
@@ -138,6 +140,7 @@
 -brand-name-mac = macOS
 -brand-name-mac-short = Mac
 -brand-name-windows = Windows
+-brand-name-windows-10 = Windows 10
```

Comment:
This is used for the subnav, and is based off the copy document. Let me know if we want to just go with "Windows".

Reply:
It's tempting but there's a chance this could get complicated with version numbers over time. I think sticking to just `Windows` for a brand name is probasbly sufficient for now?

Reply:
Here's the text from the Windows page. How should I handle this for l10n'ers? 

```
foo = { -brand-name-mozilla-vpn } for { -brand-name-windows } 10
```

<img width="656" alt="image" src="https://user-images.githubusercontent.com/2692333/110814302-eff80180-824e-11eb-9844-c565a6c50851.png">

Reply:
this format looks fine to me

Reply:
I swapped the subnav for just "Windows" and will make it Windows 10 only on the page where applicable.